standard/standard_3.php

```diff
@@ -759,7 +760,11 @@ function long2ip(int $ip): string|false {}
  * is provided, or false on an error.
  */
 #[Pure]
-function getenv(?string $name, bool $local_only = false): array|string|false {}
+function getenv(
+    #[PhpStormStubsElementAvailable(from: '5.3', to: '7.0')] $varname,
+    #[PhpStormStubsElementAvailable(from: '7.1')] ?string $name = null,
+    bool $local_only = false
```

Comment:
Hm, `$local_only` was introduced in 5.6, so from 5.3 till 5.5 we can consider it as unavailable.

Reply:
Good catch! More over according to https://www.php.net/manual/en/function.getenv.php `7.0.9 | The local_only parameter has been added.` so there is a bug in docs as well. I'll update stubs to `#[PhpStormStubsElementAvailable(from: '5.6')] bool $local_only = false`